cmd/adapter-rest/startcmd/start.go

```diff
@@ -271,9 +280,20 @@ func addRPHandlers(parameters *adapterRestParameters, router *mux.Router) error
 		return err
 	}
 
+	datasource, err := initDB(parameters.dsn)
+	if err != nil {
+		return err
+	}
+
+	// TODO init OIDC stuff in iteration 2
```

Comment:
Do we have any github issue for this TODO ?

Reply:
done